src/mariadb/mod.rs

```diff
@@ -27,10 +27,37 @@ const TAG: &str = "11.3";
 /// [`MariaDB docker image`]: https://hub.docker.com/_/mariadb
 #[derive(Debug, Default, Clone)]
 pub struct Mariadb {
-    /// (remove if there is another variable)
-    /// Field is included to prevent this struct to be a unit struct.
-    /// This allows extending functionality (and thus further variables) without breaking changes
-    _priv: (),
+    copy_to_sources: Vec<CopyToContainer>,
+}
+
+impl Mariadb {
+    /// Registers sql to be executed automatically when the container starts.
+    ///
```

Comment:
Can we mention that the method has append semantic? (in others too, please)

```suggestion
    /// Registers sql to be executed automatically when the container starts.
    /// You can call the method multiple times to add multiple scripts, they will be added and not overridden.
    ///
```

Reply:
I think the wording here can be simplified a bit, but otherwise entirely correct 😉
(I am okay with changing this to your version, if you like)

```suggestion
    /// Registers sql to be executed automatically when the container starts.
    /// You can call the method multiple times to add (not override) scripts.
    ///
```